spyder/plugins/editor/plugin.py

```diff
@@ -1473,7 +1473,7 @@ def update_code_analysis_actions(self):
         results = editor.get_current_warnings()
         # Update code analysis buttons
         state = CONF.get('lsp-server', 'code_completion') \
-                and results is not None and len(results)
+            and results is not None and len(results)

```

Comment:
We could maybe not use `\` (It is a bit unreadable)

Perhaps we could use?

```python
        state = (CONF.get('lsp-server', 'code_completion')
                 and results is not None and len(results)
                 and results is not None and len(results))
```

Thoughts @spyder-ide/core-developers ?

Reply:
I will change it!

Reply:
> We could maybe not use \ (It is a bit unreadable)

Yes, I think we (including @ccordoba12 ) all agree on this and have been instructing contributors accordingly. Its also strongly discouraged in PEP 8 (and was even proposed to be removed from the language at some point)

> Perhaps we could use?

PEP 8 recommends prefixing rather than suffixing the operator here (i.e. the first option, which is likely why linters prefer it), and I find it much more readable myself (its what I use); I'm not sure why you find it less readable at least in this case since the ``and`` operators line up rather than being awkwardly stuck on the end, while the first statement doesn't parallel with the rest anyway. However, IIRC @ccordoba12 prefers suffixing it at least for ``+``, etc himself. As far as I'm aware; it has been left up to the discretion of the contributor.